src/main/java/com/laborhelper/enterprise/dto/LaborAction.java

```diff
@@ -5,15 +5,25 @@
 import java.util.Date;
 import java.util.Map;
 
+/*
+Maybe separate this class in multiple other classes?
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.